elements/rh-table/rh-table-lightdom.css

```diff
@@ -12,46 +11,88 @@
     var(--rh-color-border-subtle-on-light, #c7c7c7);
   --rh-table-row-background-color:
     rgb(
-        var(--rh-color-gray-10-rgb, 242 242 242) / var(--rh-opacity-40, 40%)
+        var(
+          --rh-color-gray-20-rgb,
+          224 224 224
+        )
+        / var(--rh-opacity-40, 40%)
+      );
+  --rh-table-column-background-color:
+    rgb(
+        var(
+          --rh-color-blue-50-rgb,
+          0 102 204
+        )
+        / var(--rh-opacity-10, 10%)
+      );
+}
+
+rh-table[color-palette^="dark"] table {
+  --rh-table-row-background-color:
+    rgb(
+        var(
+          --_rh-color-white-rgb, /* making private as currently doesn't exist */
+          255 255 255
+        )
+        / var(--rh-opacity-10, 10%)
```

Comment:
Background color and opacity look correct here, but the Figma and the demo colors look different.

**Dark contexts**
Demo:
![Screenshot 2024-01-03 at 4 38 59 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/95588923/746b2930-5d21-4912-9c01-bc6c78126328)

Figma:
![Screenshot 2024-01-03 at 4 39 11 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/95588923/16e36166-1039-445d-b05b-a048f4cac22b)

**Light and lighter contexts**
Demo:
<img width="1059" alt="Screenshot 2024-01-03 at 4 56 28 PM" src="https://github.com/RedHat-UX/red-hat-design-system/assets/95588923/b17a1b40-fb13-4829-ab0e-500813baea0a">

Figma:
<img width="915" alt="Screenshot 2024-01-03 at 4 54 45 PM" src="https://github.com/RedHat-UX/red-hat-design-system/assets/95588923/d5bcaf08-499b-44d9-b560-d6aa8973899e">

Is it possible that the opacity value for the light and lighter contexts is being used for the dark contexts and vice versa? This is what it looks like if the dark theme row hover has a 50% opacity in Figma:
<img width="759" alt="Screenshot 2024-01-03 at 4 58 07 PM" src="https://github.com/RedHat-UX/red-hat-design-system/assets/95588923/210b21d0-b7b5-47b6-9a9d-50e061ec2abb">

Reply:
This is a bug with the last update I made, thank you for catching.  Will fix

Reply:
Pushed a fix, unfortunately it has a compromise of the context provider having to be a direct parent OR the table itself has to provide the specific color palette to match when nested.  Going to write up a issues/POC about this problem for discussion.

Reply:
POC: https://github.com/RedHat-UX/red-hat-design-system/pull/1395